packages/types/src/scope.ts

```diff
@@ -89,6 +89,17 @@ export interface Scope {
    */
   setSpan(span?: Span): this;
 
+  /**
+   * Returns the current set span if there is one
+   */
+  getSpan(): Span | undefined;
```

Comment:
Do we still want `getSpan` or is this a left over?

Reply:
We still want it, right now we don't have a case in JS where we need it but I think as soon as we start thinking about adding ORM (database) support to node we'll have the same need as python has right now.

Reply:
That's interesting. Last time I looked at the Python SDK, it only had one slot to store a "span" in the scope and it is either a span or transaction.

If we want to have a slot for both and distinguish what `getSpan` and `getTransaction` do, we need to change the scope impl.

cc @untitaker